src/query/plan/rule_based_planner.cpp

```diff
@@ -368,6 +369,12 @@ class ReturnBodyContext : public HierarchicalTreeVisitor {
 
 #undef VISIT_BINARY_OPERATOR
 
+  bool PostVisit(RangeOperator &op) override {
+    bool res = op.expr1_->Accept(*this);
+    res |= op.expr2_->Accept(*this);
```

Comment:
Does 2nd Accept need to be visited?
Can be do a short-cutting `||`

Reply:
We want to check both branches.